util/fscrypt_completion

```diff
@@ -0,0 +1,225 @@
+# bash completion for fscrypt                              -*- shell-script -*-
+
+# Output list of possible mount points
+_fscrypt_mountpoints()
+{
+    fscrypt status 2>/dev/null | \
+        command sed \
+            -e '/^\//!d' \
+            -e '/Yes$/!d' \
+            -e 's/ .*//'
+}
+
+# Output list of possible policy or protector IDs
+# $1: the mount point on which policies are looked for.
+# $2: the section (policy or protector) to retrieve
+_fscrypt_status_section()
+{
+    fscrypt status "$1" 2>/dev/null | \
```

Comment:
I'm struggling to understand these sed expressions.  I think the following would be easier to understand:

    local section=${2^^}
    fscrypt status "$1" 2>/dev/null \
            | awk '/^[[:xdigit:]]{16}/ && section == "'$section'" { print $1; next; }
                   { section = $1 }'

It also doesn't output extra blank lines.

Reply:
I'm struggling to understand these awk expressions, too. :-)  I may use yours if you prefer, it does not matter to me if I don't have to modify it.